### PR TITLE
Update Dashboard Content

### PR DESCRIPTION
**Pull request by testerslol123**
Refer to issue #14 Pull Request

I just filling some dashboard content based on latest activity, created user and roles.
And fix some minor UI bugs when creating roles.

Currently just have idea to show that information to users.
I forgot how to add label hacktoberfest in pull request.

Thank you

CC: @Tjoosten

### CODE REVIEW

**Comment by Tjoosten** [comment]
@testerslol123 Thank you for your effort in this one. Will look into the pr tomorrow.

File: resources/views/kiosk.blade.php [hunk 1]
```diff
@@ -1,5 +1,108 @@
...
+            </div>
+            <div class="col-4">
+                <div class="text-center">
+                    <h3>Latest Created Roles</h3>
+                </div>
+                <div class="table-repsonsive">
+                    <table class="table table-sm mb-0 table-hover">
+                        <thead class="text-center">
+                          <tr>
+                            <th scope="col">Role Name</th>
```
Comment (Tjoosten) [inline resources/views/kiosk.blade.php:78]: Convent the table headings plz from `Role name` to `__('Role name')` by example

File: resources/views/kiosk.blade.php [hunk 2]
```diff
@@ -1,5 +1,108 @@
...
+                              <td>
+                                <a href="{{ kioskRoute('roles.show', $role) }}" class="text-muted text-decoration-none mr-1">
+                                    <x:heroicon-o-eye class="icon"/>
+                                </a>
+                              </td>
+                            </tr>
+                            @endforeach
+                            <tr class="text-center">
+                                <td colspan="4">
+                                    <a href="{{ kioskRoute('roles.index') }}">See more</a>
```
Comment (Tjoosten) [inline resources/views/kiosk.blade.php:99]: `See more` change requested to `__('See more')`

**Review by Tjoosten — CHANGES_REQUESTED** [review]

**Comment by Tjoosten** [comment]
Hey @testerslol123.

Ty for your effort on this one. It set is temporarily on hold because the change request. 
If you have any questions feel free to contact me. 

Kind regards, 
Tim Joosten

**Comment by testerslol123** [comment]
@Tjoosten Hi, may I know what changes need to work on with?

**Comment by Tjoosten** [comment]
Op vr 9 okt. 2020 om 07:13 schreef Danny S <notifications@github.com>

File: resources/views/kiosk.blade.php (hunk 2 above)
Comment (testerslol123) [reply resources/views/kiosk.blade.php]: Hi, currently I just push the fixing

File: resources/views/kiosk.blade.php (hunk 1 above)
Comment (testerslol123) [reply resources/views/kiosk.blade.php]: Hi, currently I just push the fixing

**Comment by testerslol123** [comment]
@Tjoosten Hi, I just push the changes, I just cast all hardcoded text to localization.
Can check if something missing? Thank you.

And don't forget to add hacktoberfest

**Review by Tjoosten — COMMENTED** [review]
Good to go!

Thank for your effort on this on one.

**Pull request merged**